### ISSUE TITLE
Cant rotate usb-c connected screen

### ISSUE THREAD

**Issue by TimmyOl** [jakehilborn/displayplacer#50]
I have a macbook pro 16" with a hp usb-c hub with 2 connected hp screens with displaylink.
When i try to rotate with both Persistance id and contextual the laptop monitor blinks and it says could not rotate screen.

The laptop screen rotates fine.

```
displayplacer "id:4128832 mode:1 scaling:off origin:(1920,0) degree:90"
Error rotating screen 4128832
```

**Comment by TimmyOl** [comment]
Im sorry for the lack of information but im used to linux and dont know what you need for logs etc to look in to this?

**Comment by jakehilborn** [comment]
Please run `displayplacer list` and paste that output here.

**Comment by maurelian** [comment]
I have a similar issue, so I'll hijack this thread.
I want to have a setting for rotating my external monitor. The monitor connect to a dell doc via hdmi, which connects to my mbp via usb-c. The mbp is running macOS 11.1 (Big Sur), with an M1 chip.

This is the output of `displayplacer list` with my standard setup: 

Command 1: `displayplacer "id:37D8832A-2D66-02CA-B9F7-8F30A301B230 res:1440x900 hz:60 color_depth:8 scaling:on origin:(0,0) degree:0" "id:662F1DC9-C622-69F7-0857-D6964E3302DB res:1920x1080 hz:30 color_depth:8 scaling:on origin:(-1920,0) degree:0"`

This is the output from the arrangement with my external monitor rotated:

Command 2: `displayplacer "id:37D8832A-2D66-02CA-B9F7-8F30A301B230 res:1440x900 hz:60 color_depth:8 scaling:on origin:(0,0) degree:0" "id:662F1DC9-C622-69F7-0857-D6964E3302DB res:1080x1920 hz:30 color_depth:8 scaling:on origin:(1440,0) degree:270"`

When I run the command to rotate the monitor (command 2), I get the message
> Error rotating screen 662F1DC9-C622-69F7-0857-D6964E3302DB
Screen ID 662F1DC9-C622-69F7-0857-D6964E3302DB: could not find res:1080x1920x30 color_depth:8 scaling:off

Alternatively, I can rotate the monitor manually, and running command 2 will work. 

Basically, it seems like it's the attempt to rotate the monitor that causes it to fail.

**Comment by cvcore** [comment]
I met exactly the same issue

**Comment by bkmartin** [comment]
I am seeing this same issue with a Thunderbolt 3 hub connected to a M1 macbook pro.   It seems like the program does not recognize the target esolution as valid prior to the rotation.  The net result is you need to rotate the monitors using system preferences and displayplacer will change resolutions, scaling, refresh rate just fine, but it will not rotate.

**Comment by jakehilborn** [comment]
@TimmyOl @maurelian @cvcore @bkmartin I've switched to a different rotation API which should fix this issue. It's included in [release v1.3.0](https://github.com/jakehilborn/displayplacer/releases). Could you try it out and let me know if it works for you?
